pkgs/os-specific/linux/raspberrypi-eeprom/default.nix

```diff
@@ -1,15 +1,21 @@
 { stdenvNoCC, lib, fetchFromGitHub, makeWrapper
 , python3, binutils-unwrapped, findutils, kmod, pciutils, libraspberrypi
 }:
+
+let verinfo = {
+  version = "2022-04-26";
```

Comment:
unstable label dropped intentionally. They don't tag the repo reliably, the stable builds are just published into the stable/ directory which is what my script picks out of.